org.eclipse.jdt.ui/ui/org/eclipse/jdt/internal/ui/util/CleanUpCoreWrapper.java

```diff
@@ -48,209 +25,16 @@
  * @since 3.31
  */
 public class CleanUpCoreWrapper {
```

Comment:
I couldn't find these static `wrap(..)` calls used anywhere, so let me know if they're needed, or this entire class can be removed.